colour/models/rgb/dataset/eci_rgb_v2.py

```diff
@@ -88,55 +87,27 @@
 """
 
 
-def _eci_rgb_v2_OECF(value):
+def _scale_domain_0_100_range_0_1(value, callable_):
     """
-    Defines the *ECI RGB v2* colourspace opto-electronic conversion function.
+    Scales the domain of given callable to [0, 100] and its range to [0, 1].
```

Comment:
Just updated the docstring on that one to make what it does more explicit. I'm not sure we need a decorator as it is just a one shot private isolated function, should we have the need for it elsewhere, we will definitely move it to the `utilities` package and turn it into a proper decorator.